src/anubhav/Shop.java

```diff
@@ -0,0 +1,150 @@
+/*Assignment  7  : 30th March 2021 
+
+1. Create a system (Shop) which maintains each item's quantity.
+a. Initial stocks of Maggie: 50 Maggie packets 
+b. Initial stocks of Dosa : 43 packets 
+c. Initial stocks of pouches : 39 oil packets 
+d. Initial stocks of panipuri : 43 packets  
+e. Initial stocks of masala�s : 73 packets.
+
+Expectation:
+1) Only one Customer came for shopping, he picked different items with different quantities, 
+	print "running out of stock" message if a particular item is sold out otherwise deduct requested quantity from total quantity. 
+	Partial selling is allowed.  
+2) Create a method which shows only those item which is "Out of stock".
+3) Create a method which shows only those item which is "Available in stock" & respective quantities.*/
+
+
+package anubhav;
+
+public class Shop {
+	
+	String maggi;
+	int maggiQuantity;
+	String dosa;
+	int dosaQuantity;
+	String oilPouch;
+	int oilPouchQuantity;
+	String panipuri;
+	int panipuriQuantity;
+	String masala;
+	int masalaQuantity;
+	
+	void setInitialQuantity (int maggiQ, int dosaQ, int oilPouchQ, int panipuriQ, int masalaQ){
+		maggiQuantity= maggiQ;
+		dosaQuantity= dosaQ;
+		oilPouchQuantity= oilPouchQ;
+		panipuriQuantity= panipuriQ;
+		masalaQuantity= masalaQ;
+	}
+	
+	void quantityPicked (int maggiPicked, int dosaPicked, int oilPouchPicked, int panipuriPicked, int masalaPicked) {
+		if (maggiPicked>0){
+			maggiQuantity= maggiQuantity-maggiPicked;
+			if (maggiQuantity <0) {
+				System.out.println("Insufficient maggi stock available");

```

Comment:
Why we printing the OOS statements here, we have separate requirement for this FR2 and FR3